api/.env

```diff
@@ -0,0 +1,3 @@
+DB_IP=mongodb://127.0.0.1:27017/snake
+DB_IP_TEST=mongodb://127.0.0.1:27017/snake_test
+PORT=5550
```

Comment:
ahhhh This makes a lot more sense now this was setting the `DB_IP`.
Can you just change `DB_IP` -> `DB_URI`?